kevinbooth/connect-four
# Add rematch capability and name input statement validation

In this pull request, I added rematch functionality and name input validation for null values.

To Test:
Play a game and try to have a rematch
Play a game and try to not have a rematch by quitting
Play a game and test the validation on the rematch section
Input an empty value for your name and make sure the game catches this
